agent/src/beerocks/slave/agent_ucc_listener.cpp

```diff
@@ -18,7 +18,7 @@ using namespace net;
 
 agent_ucc_listener::agent_ucc_listener(uint16_t port, const std::string &vendor,
                                        const std::string &model, const std::string &bridge_iface,
-                                       Socket **controller_sd)
+                                       SocketClient **controller_sd)
```

Comment:
Change is correct, but the commit message is not entirely correct: it's an upcast from `SocketClient*` to `Socket*` - the additional pointer is the problem.